src/uu/test/src/parser.rs

```diff
@@ -121,11 +121,7 @@ impl Parser {
     /// Test if the next token in the stream is a BOOLOP (-a or -o), without
     /// removing the token from the stream.
     fn peek_is_boolop(&mut self) -> bool {
-        if let Symbol::BoolOp(_) = self.peek() {
-            true
-        } else {
-            false
-        }
+        matches!(self.peek(), Symbol::BoolOp(_))
```

Comment:
use of unstable library feature 'matches_macro'

Reply:
Oh.
Before I revert this, is a bump from Rust 1.40.0 to 1.42.0 planned any time soon?

Reply:
afaik, there isn't a driver for a rustc bump, sorry